torch/_torch_docs.py

```diff
@@ -2701,6 +2701,43 @@ def merge_dicts(*dicts):
     input (Tensor): the PyTorch tensor to test
 """)
 
+add_docstr(torch.is_nonzero,
+           r"""
+is_nonzero(input) -> (bool)
+
+Returns True if the :attr:`input` is a scalar which is not equal to zero
```

Comment:
is_nonzero only accepts tensor inputs, right? this suggests it can take scalar inputs.

Reply:
Yes, I'll fix that, however please decide on the naming also